vuepress/docs/next/tutorials/samples/customer-portal.md

```diff
@@ -0,0 +1,125 @@
+2. From the navigation bar, go to Users. Click `Add User` at right.
+3. Enter the fields as needed but note the requirements for the following:\
+       `Username`: A unique name\
+       `Email`: must use the same address used in Jira\
+       `User Enabled` → On\
+       `Email Verified` → Off\
+    Save
+
+4. From the headings, go to `Credentials`. \
+Under `Credential Reset`, in the `Reset Actions` → Update Password. \
+For external users, you can extend the `Expires In` setting to a minimum of 2 days. \
+Click `Send Email`. \
+This will send an email to the user to activate their account and set a new password.
+5. Role Mappings 
+For each user, select the appropriate `Realm Roles` from the `Available Roles` and click `Add Selected` to assign. If you want to use the Customer Portal default roles, choose `entandodemo-customerportal-server` from the `Client Roles` pull-down menu. This will give you additional roles that you can add. Check the Effective Roles column to make sure the correct roles have been assigned.
+
+![Entando ID Management Role Mapping](./images/custportal-identity-userrole.png)
+
+7. Under Groups, assign to appropriate groups. Roles are additive as in RBAC models.
+
+### Customer Portal Administrator
+
+As administrator for the Customer Portal, you can create and manage customers, projects and subscriptions. You will also be able to assign projects to users who have activated their account on Jira  and have been assigned roles in the Entando Identity Management System. 
+
+The administrator also needs a Jira account and the correct role assignment in the Entando ID management system.
+
+![Customer Portal Landing Page](./images/custportal-landing-page.png)
+
+**A. Create a Customer or Partner**\
+Creating a Customer or Partner is a similar process. Here we have laid out the details for adding a Customer. Please follow the same procedure for adding a Partner.
+1. Click Add a Customer
+2. Enter the details:
+     * The Customer Number must be unique. 
+     * The notes field is visible only to Support and Admin users.
+     * Click Save 
+
+![Customer Portal Add Customer](./images/custportal-addcustomer.png)
+
+**B. Create & Assign Projects**
+1. From the landing page, click on any customer to see their associated Project list.
+2. To create a new project, click on the header → Add a Project.
+    * Enter the Project details
+    * Provide the Organization ID from Jira. Only one project in the portal can be used with that ID.
+    * Click Save
+3. To assign projects
+	* Use the drop-down menu under Action on the right side and choose → 
+    Manage Users
+    * Find the user and click Submit
+Use the drop-down menu under Action to manage Partners, and request and manage subscriptions. 
+
+### User Reference
+
+All Customer Portal users need Jira Service Management credentials and Access provided by your organization administrator. You can request and track subscriptions and create and track service tickets.
```

Comment:
Access capitalized? or lowercase?
you can request and track, or the administrator can?

Reply:
mistake, lowercase.
yes, the users can too